app/app.go

```diff
@@ -547,6 +554,23 @@ func (app *Evmos) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.Res
 	return app.mm.EndBlock(ctx, req)
 }
 
+// We are intentionally decomposing the DeliverTx method so as to calculate the transactions per second.
+func (app *Evmos) DeliverTx(req abci.RequestDeliverTx) (res abci.ResponseDeliverTx) {
+	defer func() {
+		// TODO: Record the count along with the code and or reason so as to display
+		// in the transactions per second live dashboards.
+		if res.Code != 0 {
+			// TODO: Perhaps record this as a failed transaction?
```

Comment:
yes we should record them as failed imo

Reply:
Yeah I'll expand with more PRs to get failed ones too.